NLSY/nlsy_imputation.qmd

```diff
@@ -210,6 +206,12 @@ ggplot(pnetworth, aes(x=pnetworth, fill= imputation)) +
     geom_density(alpha = 0.5) +
     facet_wrap(~imputation, scales="free")
 ```
+```{r}
+ggplot(dad_educ, aes(x=dad_educ, fill= imputation)) +
```

Comment:
I realized from looking at these charts, that father's education is not an ordered factor. I think that we should change that, esp. since we specify in the imputation model that education variables are ordered. You can use a new function, `nlsy_encode_educ5`, from my latest PR, to create education as ordered factors.

Also, it may be better to color-code education levels rather than the imputation variable. That way we avoid the problem of showing education levels on the X-axis.

Reply:
Hi Damir, thanks! I have updated the old `nlsy_encode_educ4` and factor attempt with `nlsy_encode_educ5`. While I see that the encoding shows up if I generate a basic table and if I order the data frame, I think ggplot overrides that with alphabetical ordering, so I coded the levels into the distribution plots. I also flipped the x and y axis so that labels are not cut off anymore.